.eslintrc.yml

```diff
@@ -11,7 +11,7 @@ env:
 
 parserOptions:
   sourceType: module
-  ecmaVersion: 6
+  ecmaVersion: 8
```

Comment:
This is needed because somewhere in tests we uses trailing comma in function call.

Reply:
that means it should be changed in an overrides, just for tests - but not for the whole repo.

Reply:
overrides is unavailable in eslint <=3, and the Travis script run lint on all versions.

Reply:
in that case, a nested config file inside the `test` directory would work in all eslint versions.

Reply:
I chose not to do to that because then I need do that in these 5 directories `tests`, `utils`, `resolvers`, `config`, `memo-parser`. But maybe that's not that bad, what do you think?

Reply:
that’s a fair point - maybe we could make a separate file at the root, `.eslintrc-test` and then in those 5 dirs, use `extends: [‘../.eslintrc-test’]`?

Reply:
That is a good idea.